shared/actions/chat/index.js

```diff
@@ -1167,6 +1183,30 @@ function* _markThreadsStale(action: Constants.MarkThreadsStale): SagaGenerator<a
   yield put(Creators.loadMoreMessages(selectedConversation, false))
 }
 
+function* _inboxSynced(action: Constants.InboxSynced): SagaGenerator<any, any> {
+  const {convs} = action.payload
+  const author = yield select(usernameSelector)
+  const items: List<Constants.InboxState> = Shared.makeInboxStateRecords(author, convs)
+
+  const convIDs = items.reduce(function(l, item) {
```

Comment:
simpler as a map
```
const convIDs = items.map(item => item.conversationIDKey)
```

Reply:
That creates ` List` though, I want a plain old array.

Reply:
I guess I can just call `toArray`

Reply:
yeah, or call toArray first